pandas/io/excel.py

```diff
@@ -40,17 +40,16 @@
 _writers = {}
 
 _read_excel_doc = """
-Read an Excel table into a pandas DataFrame
+Read an Excel table into a pandas DataFrame.
 
 Parameters
 ----------
-io : string, path object (pathlib.Path or py._path.local.LocalPath),
-    file-like object, pandas ExcelFile, or xlrd workbook.
+io : str, path object (pathlib.Path or py._path.local.LocalPath), \
+     file-like object, pandas ExcelFile, or xlrd workbook
     The string could be a URL. Valid URL schemes include http, ftp, s3,
     gcs, and file. For file URLs, a host is expected. For instance, a local
-    file could be file://localhost/path/to/workbook.xlsx
-sheet_name : string, int, mixed list of strings/ints, or None, default 0
-
+    file could be file://localhost/path/to/workbook.xlsx.
```

Comment:
I'd remove this last paragraph, as may confuse users (`/path/to/workbook.xlsx` is the normal way of opening a local file, not `file://localhost/path/to/workbook.xlsx`)

Reply:
Gone!